lib/src/sv_internal.h

```diff
@@ -32,6 +32,10 @@
 #include "legacy_validation.h"  // legacy_sv_t
 #include "sv_defines.h"  // svrc_t, sv_tlv_tag_t
 
+#ifndef HAS_ONVIF
+// If ONVIF is missing, define it as an alias to signed_video_t
+typedef struct signed_video_t onvif_media_signing_t;
```

Comment:
It might be better to depend on something more standard like `void` instead of `struct signed_video_t`